docs/source/examples/tutorial/inductive_lp/semi_inductive_dataset.py

```diff
@@ -0,0 +1,19 @@
+"""Creating a semi-inductive dataset."""
+
+import logging
+
+from pykeen.datasets import get_dataset
+from pykeen.datasets.base import EagerDataset
+
+logging.basicConfig(level=logging.INFO)
+
+# we use all of CodexSmall's data as source graph
+dataset = get_dataset(dataset="CodexSmall")
+dataset.summarize()
+assert dataset.validation is not None
+tf_all = dataset.training.merge(dataset.validation, dataset.testing)
```

Comment:
I think it would be nice to have a Dataset class-level function that calls this line, as a high-level abstraction

Reply:
https://github.com/pykeen/pykeen/pull/1416/commits/c47b033c1a5f372e9e721855d8fe378213835fca